.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish docker image
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - master         # Push events on master branch
+
+jobs:
+  deploy-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout devito
+        uses: actions/checkout@v1
+
+      - name: Check event name
+        run: echo ${{ github.event_name }}
+
+      - name: CPU image
+        if: github.event_name == 'push'
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: mloubout/devito
+          dockerfile: ./docker/Dockerfile
```

Comment:
eventually, we'll add another one for Dockerfile.nvidia right?

Reply:
yep the one fro #1364 will be added

Reply:
added